src/win/handle.c

```diff
@@ -157,3 +157,7 @@ int uv_is_closing(const uv_handle_t* handle) {
 uv_os_fd_t uv_get_osfhandle(int fd) {
   return uv__get_osfhandle(fd);
 }
+
+int uv_open_osfhandle(uv_os_fd_t os_fd) {
+  return _open_osfhandle((intptr_t) os_fd, 0);
```

Comment:
Are there any implications to only passing `0` as the flags?

Reply:
It only has meaning when someone is querying the flags of an FD. I'm not aware of any win32 function to query the open mode of a handle.